src/stories/theme/_list.css

```diff
@@ -1,3 +1,41 @@
+/**
+ * [1] - Due to PostCSS limitations in CKEditor we must include our icon
+ *       directly.
+ */
+.ck.ck-list__item {
+  & .ck-button {
+    position: relative;
+    padding: 10px 32px;
+
+    &::before {
+      position: absolute;
+      top: 0;
+      left: 0;
+      transition: opacity .1s ease-in-out;
+      opacity: 0;
+      background: no-repeat 50% / 16px url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' focusable='false' viewBox='0 0 16 16' color='%231f73b7'%3E %3Cpath fill='none' stroke='currentColor' stroke-linecap='round' stroke-linejoin='round' d='M1 9l4 4L15 3'/%3E %3C/svg%3E"); /* [1] */
```

Comment:
I recall `data:` URL's were avoided in some of the migrations in `react-components`. Just wanted to confirm this is okay?

Reply:
Agreed that these need to be avoided. Right now I'm trying to align with the formats provided in the CK theme files since it goes through their PostCSS config. Once we are able to run it through our own PostCSS process we will be able to extract CSS variables as well as use `svg-load` for this area. [CK example](https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-theme-lark/theme/ckeditor5-media-embed/mediaembedediting.css#L54)

Reply:
Gotcha thanks for the explanation!